coreth.go

```diff
@@ -62,9 +61,8 @@ func NewETHChain(config *eth.Config, nodecfg *node.Config, etherBase *common.Add
 	//}
 	cb := new(dummy.ConsensusCallbacks)
 	mcb := new(miner.MinerCallbacks)
-	bcb := new(eth.BackendCallbacks)
-	backend, _ := eth.New(node, config, cb, mcb, bcb, chainDB, settings)
-	chain := &ETHChain{backend: backend, cb: cb, mcb: mcb, bcb: bcb}
+	backend, _ := eth.New(node, config, cb, mcb, chainDB, settings)
```

Comment:
Why don't we check this error?

Reply:
Never has been before, but will update now to panic if there's an error here.